reana_server/reana_admin.py

```diff
@@ -214,39 +218,102 @@ def import_users(ctx, admin_access_token, file_):
 @admin_access_token_option
 @click.option(
     '--id',
+    'id_',
     help='The id of the user.')
 @click.option(
     '-e',
     '--email',
     help='The email of the user.')
-def token_grant(id, email, admin_access_token):
-    """."""
-    user_id = 'dc4aef21-c324-4fd5-8480-c724de1f0c06'
-    user_email = 'john.doe@google.com'
-    user_granted_token = 'c0fa47fa00ae4013a13fd7n'
-    # create a token in UserToken table and activate it
-    # create an audit log entry
-    # send notification to user by email
-    click.secho(f'Token for user {user_id} ({user_email}) granted.\n'
-                f'\nToken: {user_granted_token}', fg='green')
+@click.option(
+    '-f',
+    '--force',
+    is_flag=True,
+    default=False,
+    help='Grant token even if user has a revoked access token.')
+def token_grant(admin_access_token, id_, email, force):
+    """Grant a token to the selected user."""
+    try:
+        admin = User.query.filter_by(id_=ADMIN_USER_ID).one_or_none()
+        if admin_access_token != admin.access_token:
+            raise ValueError('Admin access token invalid.')
+        user = _get_user_by_criteria(id_, email)
```

Comment:
What do you think of adding `_get_user_by_criteria` to be the callback that loads the user so `--user` could be `id` or `email`? Also, this callback could be giving an error if the user doesn't exist, so we don't have to duplicate this error handling.

Reply:
Seeing the `elif`'s complexity, previous comment isn't so important as more conditions are chained and cannot be reused from the two commands...